core/settings.py

```diff
@@ -255,15 +254,15 @@ def expose_site_context(request):
 CRISPY_TEMPLATE_PACK = "bootstrap4"
 
 # Email configuration (production uses Mailjet - see README)
-SEND_IN_BLUE_API_KEY = get_env_variable("SEND_IN_BLUE_API_KEY")
+SEND_IN_BLUE_API_KEY = env("SEND_IN_BLUE_API_KEY")
 
 EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
 EMAIL_USE_TLS = True
 EMAIL_USE_SSL = False
-EMAIL_HOST = get_env_variable("EMAIL_HOST")
-EMAIL_PORT = get_env_variable("EMAIL_PORT", type=int)
-EMAIL_HOST_USER = get_env_variable("EMAIL_HOST_USER")
-EMAIL_HOST_PASSWORD = get_env_variable("EMAIL_HOST_PASSWORD")
+EMAIL_HOST = env("EMAIL_HOST")
+EMAIL_PORT = env.int("EMAIL_PORT")
+EMAIL_HOST_USER = env("EMAIL_HOST_USER")
+EMAIL_HOST_PASSWORD = env("EMAIL_HOST_PASSWORD")
 DEFAULT_EMAIL = "acceslibre@beta.gouv.fr"
 DEFAULT_FROM_EMAIL = f"L'équipe {SITE_NAME} <{DEFAULT_EMAIL}>"
 MANAGERS = [("Acceslibre", DEFAULT_EMAIL)]
```

Comment:
I think we should also use this PR to remove https://pypi.org/project/dj-database-url/ and use something like `**env.db("DATABASE_URL")` to get the URL right

Reply:
And same goes for cache I think